ringo_printtemplate/views.py

```diff
@@ -78,9 +79,11 @@ def save_file(request, item):
         request.POST.get('file').file.seek(0)
         data = request.POST.get('file').file.read()
         filename = request.POST.get('file').filename
+        item.mime = check_mime_type_from_buffer(data)
+        if item.mime != "application/vnd.oasis.opendocument.text":
+            raise AttributeError()
```

Comment:
Again, this save_file method is only called if the validator did not fail. So there is no reason for this explicit check here.